enki/PhysicalEngine.h

```diff
@@ -393,6 +396,8 @@ namespace Enki
 		virtual void doGlobalInteractions(double dt, World* w);
 		//! Sort local interactions. Called by addLocalInteraction ; can be called by subclasses in case of interaction radius change.
 		void sortLocalInteractions(void);
+
+		virtual void clickedInteraction(bool pressed, int buttonCode, double pointX, double pointY, double pointZ){};
```

Comment:
I just need it for robot so I put it here, but if you want to move it in PhysicalObject, why not.
What do you mean by bit field ?